lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -566,6 +568,20 @@ def _get_rule_id(self, flow):
             self.logger.error('Could not find rule id for num %d: %s',
                               rule_num, e)
             return ""
+            
+    def get_stats(self, cookie: int = 0, cookie_mask: int = 0):
+        _, parser = self._datapath.ofproto, self._datapath.ofproto_parser
+        message = parser.OFPFlowStatsRequest(datapath=self._datapath, cookie = cookie, cookie_mask = cookie_mask)
+        try:
+            response = ofctl_api.send_msg(self, message, reply_cls=parser.OFPFlowStatsReply,
+                    reply_multi=True)
+            RRTable = self._get_usage_from_flow_stat(response[0].body)
```

Comment:
change name to confirm to style standards, smth like `flows = self._get_usage_from_flow_stat(response[0].body)`